static/docs/commands-reference/status.md

```diff
@@ -53,17 +54,30 @@ be run again if `dvc repro` were executed.
 If instead, differences are detected, `dvc status` lists those changes. For each
 DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
 differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+shown, and additionally, a status word is shown describing the changes in
+checksum or status for both local and cloud workspace. Given list describes
```

Comment:
Given list sounds like a reference to the listed items above, not to the list below